awx/main/migrations/0024_v330_add_oauth_activity_stream_registrar.py

```diff
@@ -55,21 +55,6 @@ class Migration(migrations.Migration):
                 'verbose_name': 'access token',
             },
         ),
-        migrations.CreateModel(
```

Comment:
Do we need a similar migration to create the table for the native `RefreshToken` ?

Reply:
It looks like Django-oauth-toolkit handles that on it's own.  and the model shows up in the database.  
![image](https://user-images.githubusercontent.com/11698892/36866337-02351a88-1d60-11e8-8884-d32e9f714009.png)